carbon-design-system/carbon-addons-iot-react
# [DashboardEditor] 1817 image edit form

Closes #1817 

**Summary**

- This change sets values for Image card based off the form inputs. 

**Change List (commits, features, bugs, etc)**

- Added Image settings form
- Changed styling for dashboard grid  to allow for padding between grid and form

**Acceptance Test (how to verify the PR)**

- Go to https://deploy-preview-1824--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-experimental-dashboardeditor--default 
- Add image card. You should be able to change aspects to card  accordingly.
